### DISCUSSION TITLE
Beta For you tab layout issues in the desktop view

### DISCUSSION THREAD

**Discussion by ferhatkefsiz** [community/community#55287] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

There is a layout issue with "For you" tab of the feed tab. See screenshot:

<img width="847" alt="Screenshot 2023-05-12 at 13 20 49" src="https://github.com/community/community/assets/72281546/40916dbc-53ee-4dcb-a8d0-ef3c6ea86f68">

**Comment by MansiAyer** [comment]
Same issue on my feed, it isn't there on the following tab though- just the "for you" tab
